### DISCUSSION TITLE
Improvement: New Light Mode

### DISCUSSION THREAD

**Discussion by ThGnommy** [ArslanYM/StarterHive#231] [Ideas]
Hi, with @ArslanYM in a prev [PR](https://github.com/ArslanYM/StarterHive/pull/223) we have thought about remaking the Light Mode. 

I also noticed that by analyzing the page with Lighthouse there is poor colour contrast.

<img width="1512" alt="image" src="https://github.com/ArslanYM/StarterHive/assets/29650993/52e7f7d6-feff-40a6-bb4d-08d8eb4440fb">

We could discuss more some new colours we can use or any different approach 🙂

**Comment by ArslanYM** [comment]
Youre right @ThGnommy , Do you have any ideas in mind for a deep purple look in the light mode.

**Reply by ThGnommy** [reply]
I thought we could add a CSS filter to the images when the theme is in light mode? I'm not a fan of this but it could be an option. Or find a darker yellow colour for the buttons, the 'issue' and the buttons in the navbar.

**Reply by ArslanYM** [reply]
I have no idea how that would look. Any mockups I can refer to?

**Reply by ThGnommy** [reply]
Something like this, I used the hue-rotation for all the images on the homepage

<img width="1511" alt="image" src="https://github.com/ArslanYM/StarterHive/assets/29650993/c263d428-2631-4317-8182-ecca6437bf75">

But I'm not really sure. Does it look weird?

About the buttons and navbar links, I tried to check the colour contrast with the tailwind color `bg-yellow-500` and `bg-yellow-600` and nothing pass the tests. I used this [site](https://dequeuniversity.com/rules/axe/4.6/color-contrast) 

I also come up with this:

Normal:
<img width="939" alt="image" src="https://github.com/ArslanYM/StarterHive/assets/29650993/71a441f4-3602-4181-8fb1-9e1ed01626e6">

Button hovered:
<img width="936" alt="image" src="https://github.com/ArslanYM/StarterHive/assets/29650993/b7bfce70-3e28-429c-b530-b2b4150ed02e">


What do you think?
